mlflow/tracking/context/git_context.py

```diff
@@ -45,8 +57,18 @@ def _source_version(self):
             self._cache["source_version"] = _get_source_version()
         return self._cache["source_version"]
 
+    @property
+    def _git_repo_url(self):
+        if "git_repo_url" not in self._cache:
+            self._cache["git_repo_url"] = _get_git_repo_url()
+        return self._cache["git_repo_url"]
+
     def in_context(self):
         return self._source_version is not None
 
     def tags(self):
-        return {MLFLOW_GIT_COMMIT: self._source_version}
+        return {
+            MLFLOW_GIT_COMMIT: self._source_version,
+            MLFLOW_GIT_REPO_URL: self._git_repo_url,
```

Comment:
Do you plan to add `MLFLOW_GIT_BRANCH` in a follow up PR?

Reply:
Added it to this one! Thanks :)